tests/validation/validate_ball_and_stick.hpp

```diff
@@ -50,7 +50,7 @@ void run_ncomp_convergence_test(
                 seg->set_compartments(ncomp);
             }
         }
-        hw::node nd(1, backend==backend_kind::gpu? 1: 0);
+        hw::node_info nd(1, backend==backend_kind::gpu? 1: 0);
```

Comment:
Just a matter of taste, I guess, but I don't really see `a==b? 1: 0` to be any clearer than just `a==b` in these contexts.